hooks/pre/30-upgrade.rb

```diff
@@ -102,6 +102,25 @@ def mark_qpid_cert_for_update
   end
 end
 
+# rubocop:disable MethodLength
+def recreate_event_queue
+  # piping output to /dev/null because on EL6 any qpid-stat/config commands gives a non fatal python traceback.
+  cert = '/etc/pki/katello/qpid_client_striped.crt'
+  `qpid-config --ssl-certificate #{cert} -b "amqps://localhost:5671" del exchange event --durable > /dev/null 2>&1`
```

Comment:
I'm not super familiar with the installer code but am wondering why you use backticks over `Kafo::Helpers.execute`?